code/src/App.css

```diff
@@ -0,0 +1,449 @@
+padding-bottom: 20px;
+}
+
+.p-drop-down-box {
+  font-family: 'Poppins', sans-serif;
+  color: #37372C;
+  font-weight:bold;
+  text-align: center;
+}
+
+.pick-favorite {
+  font-family: 'Poppins', sans-serif;
+  color: #824947;
+  font-weight: bold;
+  padding: 3px;
+}
+
+.pick-field {
+  background-color:#D1CCC1;
+  color: #824947;
+  font-family: 'Monaco';
+  height: 30px;
+}
+.opt-po {
+  background-color:#D1CCC1;
+  color: #824947;
+  font-family: 'Monaco';
+  font-weight: bold;
+}
+
+
+/* SUBMIT */
+
+.sub-button-box {
+  display: flex;
+  justify-content: center;
+  padding: 20px 0 20px 0;
+
+}
+
+.submit-button {
+background-color: #D1CCC1;
+border: solid 1px #824947;
+height: 30px;
+font-family: 'Monaco';
+color: #37372C;
+font-size: 20px;
+}
+
+
+/* RESULT */
+
+.result-box {
+display:flex;
+flex-direction: column;
+align-items: center;
+width: 80%;
+margin: auto;
+}
+
+.p-result-box {
+  font-family: 'Poppins', sans-serif;
+  color: #824947;
+  font-weight: bold;
+  font-size: 25px;
+}
+
+
+.p-result-box-sum {
+  font-family: 'Monaco';
+  color: #37372C;
+  text-align: center;
+  font-size:20px;
+  font-weight: bold;
+}
+
+.ice-pic {
+  width: 300px;
+  height: 300px;
+  border: solid 1px #824947;
+  border-radius: 50%;
+  margin: 5px;
+}
+
+.again-btn-box {
+  display: flex;
+}
+
+.again-btn {
+  margin: 10px;
+  font-family: 'Monaco';
+  background-color: #AD8841;
+  color: #37372C;
+  border: solid 1px #824947;
+  height: 30px;
+}
+
+/*** MEDIA TABLETT ***/
+
+@media (min-width: 667px) {
+
+  .header-box {
+    margin-top: 10px;
+  }
+ 
+  h1 {
+    font-size: 40px;
+  }
+  .name-box {
+    width: 70%;
+  }
+
+  h2 {
+    font-size: 30px;
+  }
+   .fill-in-field {
+    height: 45px;
+    width: 180px;
+    font-size: 25px;
+   }
+
+   .p-button-box {
+     font-size: 25px;
+     width: 80%;
+     text-align: center;
+   }
+
+   .button-small-box {
+     width: 80%;
+     justify-content: space-evenly;
+     align-content: space-around;
+   }
+
+   .button-box {
+     width: 80%;
+     margin: auto;
+     font-size: 20px;
+     margin-bottom: 8px;
+     
+   }
+
+   .checkmark {
+     width: 25px;
+     height: 25px;
+   }
+   
+   .button-box .checkmark:after {
+     top: 8px;
+     left: 8px;
+   }
+
+   .p-drop-down-box {
+    font-size: 25px;
+    width: 70%; 
+   }
+
+   .pick-favorite {
+     font-size: 20px;
+     padding:5px;
+   }
+
+   .pick-field {
+     height: 45px;
+     font-size: 20px;
+   }
+
+   .sub-button-box {
+    margin: 25px 0 25px 0;
+  }
+
+   .submit-button {
+     height: 40px;
+     font-size: 20px;
+     
+   }
+
+   .again-btn {
+    margin: 20px;
+    height: 40px;
+    font-size: 20px;
+  }
+}
+
+/*** MEDIA DESKTOP ***/
+
+@media (min-width: 1024px) {
+
+  .header-box {
+    margin-top: 20px;
+  }
+
+  h1 {
+    font-size: 60px;
+    margin: 20px;
+  }
+
+  .name-box {
+    width: 60%;
+    margin: 20px 50px 20px 50px;
+  }
+
+  h2 {
+    font-size: 40px;
+  }
+
+  .fill-in-field {
```

Comment:
Maybe add the same structure with headings in the media queries as well? Makes it a lot easier to follow!